lib/runPrettier.js

```diff
@@ -1,22 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const { promisify } = require('util');
 const chalk = require('chalk');
 const runBin = require('./runBin');
 
-const runPrettier = ({ write, listDifferent, config, filePattern }) => {
-  const prettierConfig = config || {};
+const exists = promisify(fs.access);
+const prettierIgnorePath = path.join(process.cwd(), '.prettierignore');
```

Comment:
Should we share this between `configure` and here?

Reply:
As discussed, leaving as is in lieu of a refactor of `config/builds` making paths like these more available around the code base.